src/pages/embed.vue

```diff
@@ -228,6 +239,31 @@ onBeforeMount(() => {
     window.logixProjectId = route.params.projectId
 })
 
+onMounted(() => {
+    switch (theme.value) {
+    case 'default':
+        updateThemeForStyle('Default Theme');
+        break;
+    case 'night-sky':
+        updateThemeForStyle('Night Sky');
+        break;
+    case 'lite-born-spring':
+        updateThemeForStyle('Lite-born Spring');
+        break;
+    case 'g-and-w':
+        updateThemeForStyle('G&W');
+        break;
+    case 'high-contrast':
+        updateThemeForStyle('High Contrast');
+        break;
+    case 'color-blind':
+        updateThemeForStyle('Color Blind');
+        break;
+    default:
+        updateThemeForStyle(selectedTheme);
```

Comment:
Can we have some ENUMs const for themes
```
export const THEME = {
    high-contrast: 'High Contrast' as const,
    color-blind: 'Color Blind' as const,
.....
}
```

and in vue just call updateThemeForStyle(THEME.theme.value)

Reply:
Where should we declare the enum? I am thinking we can either put it in the `src/simulator/src/themer/themer.js` and convert the file to TypeScript, or we can create a utils folder in the src directory, i.e., `src/utils/theme.ts`, need your suggestion on that